src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -532,8 +538,16 @@ private void OutputResultsHelper(IEnumerable<CommandInfo> results)
                         }
                         else
                         {
-                            // Write output as normal command info object.
```

Comment:
There is already `count += 1;` at the top, which was not used previously. We need to move it to the bottom of the loop.